all.bash

```diff
@@ -16,6 +16,7 @@
 # limitations under the License.
 #
 
+source /etc/profile
```

Comment:
This file may not exist in some environment?
Should check existing before calling it.